app/web/features/profile/locales/en.json

```
@@ -127,37 +128,61 @@
   "report_reason": "Reason",
   "report_user": "Report this user",
   "leave_reference": {
+    "add_a_reference": "Add a reference",
     "appropriate_behavior": "Appropriate Behavior",
-    "appropriate_explanation": "Awesome –– We hope you had a great time! To help keep our community safe, we want to ask about your interaction with your fellow Coucher.",
+    "appropriate_explanation": "References keep users accountable and reward great experiences. Thank you for doing your part to help keep the community safe, trustworthy, and welcoming for everyone! ",
     "appropriate_question": "Did you feel safe with this person's behavior?",
+    "by_writing_thoughtful": "By writing a thoughtful reference, you are doing your part to keep the community a safe, trustworthy, and welcoming place for everyone!",
     "coucher_was_appropriate": "Yes, this person's behavior was appropriate.",
     "coucher_was_not_appropriate": "No, this person's behavior was not appropriate.",
     "contact_text": "If you have any questions or wish to provide additional information, please don't hesitate to <1>contact us here.</1>",
+    "did_stay_explanation": "You <strong>will not be able to write a reference</strong> if you indicate you did not meet up.\n The other member <strong>will not be notified</strong> and they can still leave a reference. If you need to report inappropriate or unwanted behavior, please report this user.",
     "host_request_reference_explanation": "Host request references are only made visible once both references have been written, or after 2 weeks. Hold tight!",
     "invalid_reference_type": "Invalid reference type",
     "invalid_step": "Invalid form step",
     "previous_step": "Previous step",
-    "private_answer": "Your answer will not be seen by the other person, and will remain private.",
+    "private_answer": "Your answers on this page are private, and will only be seen by the Couchers.org Safety Team.",
     "private_text_label": "You will also submit the following private answers:",
     "public_answer": "This will appear publicly in the References section of their profile.",
-    "public_text_label": "You are leaving the following reference on your fellow Coucher's Guestbook:",
+    "public_text_label": "You are leaving the following public reference on your fellow Coucher's Guestbook:",
```

There is nothing in the UI called "guestbook" so I don't think we should refer to it. Instead, we should refer to the person's profile because that is where the reference will appear:

```suggestion
    "public_text_label": "You are leaving the following public reference which will appear on {{name}}'s profile:",
```